### PR TITLE
Atari fix post mega merge

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

**Review by emlowe — DISMISSED** [review]
I approve even in Draft.

File: chia/data_layer/data_store.py [hunk 1]
```diff
@@ -332,9 +333,9 @@ async def get_roots_between(
                 "AND generation >= :generation_begin AND generation < :generation_end ORDER BY generation ASC",
                 {"tree_id": tree_id.hex(), "generation_begin": generation_begin, "generation_end": generation_end},
             )
-            if cursor.rowcount < 1:
-                raise Exception(f"tree id not in store {tree_id.hex()}")
             roots = [Root.from_row(row=row) async for row in cursor]
+            if len(roots) == 0:
```
Comment (almogdepaz) [inline chia/data_layer/data_store.py:337]: why not do a try catch gere as well if thats the way we are handling that case ?

Comment (altendky) [reply chia/data_layer/data_store.py:337]: How would you write the previous line to result in an exception for 0 elements but not for 1 or more while still capturing them all into `roots`?  But yes, this also left my brain a bit conflicted.  I decided not to spend a lot of time trying to come up with a solution that was good enough to act as a quality reference.  Just made something work.

Comment (almogdepaz) [reply chia/data_layer/data_store.py:337]: i think do it like in get_root and another for loop for parsing into Root, or remove the try/catch there and check the size?
seems  rn you will trow here is there is no result like in the other case

**Review by almogdepaz — DISMISSED** [review]

**Review by emlowe — DISMISSED** [review]
Ok for me - we need to get this into Atari to fix the branch

File: chia/data_layer/data_store.py (hunk 1 above)
Comment (almogdepaz) [reply chia/data_layer/data_store.py:337]: i think i get the issue, this syntax is terrible sorry

**Review by almogdepaz — DISMISSED** [review]

**Review by emlowe — DISMISSED** [review]
ok

**Review by emlowe — APPROVED** [review]
ok

**Pull request merged**